{
  try {
    return calls.call(mock,name,new Object[]{args});
  }
 catch (  Throwable throwable) {
    throw new RuntimeException(throwable);
  }
}
